go.mod

```diff
@@ -3,22 +3,17 @@ module github.com/googleforgames/open-saves
 go 1.14
 
 require (
-	cloud.google.com/go v0.84.0 // indirect
 	cloud.google.com/go/datastore v1.5.0
 	github.com/golang/mock v1.6.0
 	github.com/golang/protobuf v1.5.2
 	github.com/gomodule/redigo v1.8.5
 	github.com/google/uuid v1.2.0
 	github.com/sirupsen/logrus v1.8.1
+	github.com/spf13/viper v1.10.1
 	github.com/stretchr/testify v1.7.0
 	github.com/vmihailenco/msgpack/v5 v5.3.4
 	gocloud.dev v0.23.0
-	golang.org/x/net v0.0.0-20210614182718-04defd469f4e // indirect
-	golang.org/x/oauth2 v0.0.0-20210615190721-d04028783cf1 // indirect
-	golang.org/x/sys v0.0.0-20210616094352-59db8d763f22 // indirect
-	google.golang.org/api v0.48.0
-	google.golang.org/genproto v0.0.0-20210614182748-5b3b54cad159 // indirect
-	google.golang.org/grpc v1.38.0
+	google.golang.org/api v0.63.0
+	google.golang.org/grpc v1.43.0
```

Comment:
these too, because they're out of the scope of the PR

Reply:
These are added when getting viper.